subprojects/gradle/lint-report/src/integTest/kotlin/com/avito/android/lint/LintSlackAlertIntegrationTest.kt

```diff
@@ -8,7 +8,7 @@ import com.avito.logger.StubLoggerFactory
 import com.avito.slack.SlackClient
 import com.avito.slack.model.SlackChannel
 import com.avito.time.StubTimeProvider
-import com.avito.utils.fileFromJarResources
+import com.avito.utils.ResourcesReader
 import okhttp3.HttpUrl.Companion.toHttpUrl
 import org.junit.jupiter.api.Test
 
```

Comment:
Can we remove `com.avito.android.lint-report` plugin?

Reply:
Yes, we can; will do in separate PR